--- pidgin-2.10.1/libpurple/Makefile.am~	2011-12-06 09:44:33.000000000 +0100
+++ pidgin-2.10.1/libpurple/Makefile.am	2011-12-13 17:14:37.492942189 +0100
@@ -32,7 +32,7 @@ endif
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = purple.pc
 
-SUBDIRS = $(GCONF_DIR) plugins protocols ciphers . tests example
+SUBDIRS = $(GCONF_DIR) protocols ciphers . plugins tests example
 
 purple_coresources = \
 	account.c \
--- pidgin-2.10.1/libpurple/plugins/perl/Makefile.am~	2011-12-06 09:44:33.000000000 +0100
+++ pidgin-2.10.1/libpurple/plugins/perl/Makefile.am	2011-12-13 17:15:09.437943687 +0100
@@ -5,7 +5,7 @@ perl_dirs = common
 plugin_LTLIBRARIES = perl.la
 
 perl_la_LDFLAGS = -module -avoid-version
-perl_la_LIBADD = $(GLIB_LIBS) $(PERL_LIBS)
+perl_la_LIBADD = $(GLIB_LIBS) $(PERL_LIBS) $(top_builddir)/libpurple/libpurple.la
 perl_la_SOURCES = \
 	perl.c \
 	perl-common.c \
@@ -15,7 +15,8 @@ perl_la_SOURCES = \
 
 perl_la_DEPENDENCIES = \
 	.libs/libperl_orig.a \
-	.libs/DynaLoader.a
+	.libs/DynaLoader.a \
+	$(top_builddir)/libpurple/libpurple.la
 
 .libs/libperl_orig.a:
 	@mkdir -p .libs
