--- pidgin-2.10.0/libpurple/protocols/gg/gg.c.gg-search	2011-08-18 10:51:03.000000000 +0200
+++ pidgin-2.10.0/libpurple/protocols/gg/gg.c	2011-08-22 15:57:02.000000000 +0200
@@ -533,6 +533,9 @@ static void ggp_callback_find_buddies(Pu
 		purple_request_fields_get_string(fields, "firstname"));
 	form->nickname = g_strdup(
 		purple_request_fields_get_string(fields, "nickname"));
+	form->uin  = charset_convert(
+				purple_request_fields_get_string(fields, "uin"),
+				"UTF-8", "CP1250");
 	form->city = g_strdup(
 		purple_request_fields_get_string(fields, "city"));
 	form->birthyear = g_strdup(
@@ -586,6 +589,11 @@ static void ggp_find_buddies(PurplePlugi
 	purple_request_field_string_set_masked(field, FALSE);
 	purple_request_field_group_add_field(group, field);
 
+	field = purple_request_field_string_new("uin",
+			_("UIN"), NULL, FALSE);
+	purple_request_field_string_set_masked(field, FALSE);
+	purple_request_field_group_add_field(group, field);
+
 	field = purple_request_field_string_new("city",
 			_("City"), NULL, FALSE);
 	purple_request_field_string_set_masked(field, FALSE);
